packs/windows-iad-sami-compliance.conf

```diff
@@ -0,0 +1,218 @@
+// inspired from https://github.com/iadgov/Splunk-Assessment-of-Mitigation-Implementations/raw/master/SAMI_Business_Logic.xlsx
+// default interval 1d=86400sec
+{
+  "platform": "windows",
+  "queries": {
+    // AE
+    "win_compliance_secureboot": {
+      "query" : "select * from registry where key like 'HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Control\\SecureBoot\\State\\UEFISecureBootEnabled'",
```

Comment:
Why the `like` here? Seems like you haev the precise registry path and you could get away with `key =` here and those below.

Reply:
I think I used like everywhere, exact path or %. can be updated